src/PolynomialDerivative.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+/** <p>PolynomialDerivative calculate the derivative of input polynomial</p>
+
+    <p>PolynomialDerivative allows users input data from console or command line</p>
+
+    @author Siyao Li
+    @version UCSB, CS56, F17
+*/
+
+public class PolynomialDerivative{
```

Comment:
This class should extend `Polynomial` and include the implementation of the `derivative()` method